### ISSUE TITLE
Example fails to render anything on the map

### ISSUE THREAD

**Issue by monfera** [uber/deck.gl#71]
I did a fresh clone, recent clean install of `master`, filled in the MAPBOX_ACCESS_TOKEN with a key that I can use, enabled react-dev-tools, did
```
npm install
npm start
```
and got a lot of assertion errors, the first one being in `layer-manager.js`:
```
35     const error4 = initializeNewLayers(newLayers, {gl});
```
where error4 becomes:
```
{"name":"AssertionError","expected":true,"operator":"==","message":"Attribute updater for instancePickingColors missing update method","generatedMessage":false}
```
The page loads to the point where the list in the bottom left (Hexagon etc. Click) and the map around SF appear, but I suppose due to these errors, clicking doesn't do anything and no data shows up. 
I'm sure it's something very basic

**Comment by ibgreen** [comment]
>  clicking doesn't do anything and no data shows up

@monfera Thanks for catching this regression. Have published a fix in 2.4.8. Pull and npm install (the issue was in a dependency), and things should work again.

**Comment by monfera** [comment]
Thank you @ibgreen indeed it's a clean install and example now.
